.tekton/osc-operator-push.yaml

```diff
@@ -0,0 +1,604 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/sandboxed-containers-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
+      == "devel"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: openshift-sandboxed-containers
+    appstudio.openshift.io/component: osc-operator
+    pipelines.appstudio.openshift.io/type: build
+  name: osc-operator-on-push
+  namespace: ose-osc-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/ose-osc-tenant/osc-operator:{{revision}}
```

Comment:
There is something wrong. I'd expect it to be `quay.io/redhat-user-workloads/ose-osc-tenant/openshift-sandboxed-containers/osc-operator:{{revision}}`

If I compare to what we did with trustee, [the same PR](https://github.com/openshift/trustee/pull/2) done by the konflux bot had the right location (using the "trustee" subdir):
```
  - name: output-image
    value: quay.io/redhat-user-workloads/ose-osc-tenant/trustee/trustee:{{revision}}
```

Keeping it there for now, I'll need to check konflux settings (and maybe ask around).
Please do not merge yet.

Reply:
@spotlesstofu - this is where my experience differs from yours :-(
Asked on the konflux-user slack channel, waiting for feedback.

Reply:
Answer from the konflux people is that this is the new default.
It seems we were lucky with Trustee to get our images grouped by application. Now the default is to have them all in the same repo.
There are alternatives, and it can be modified. But for now this code is right, so we can proceed with it, and decide later about the location of images.

Reply:
Thanks for sharing! That explains why @lmilleri had to use this new URL format in the latest [trustee-fbc-4-18](https://github.com/openshift/trustee-fbc/blob/main/.tekton/trustee-fbc-4-18-pull-request.yaml#L28).